main.py

```diff
@@ -1 +1,147 @@
 # Secret Santa Chain Generator
+
+import csv
+import random
+import smtplib, ssl
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.mime.image import MIMEImage
+
+# Extracts participants, emails, and exclusions from CSV file
+with open('assets/SS-participants.csv', newline='') as f:
+    reader = csv.reader(f)
+    participant_data = list(reader)
+
+# Extracts list of participants and emails from the data list
+participants = [i[1] for i in participant_data[1:]]
```

Comment:
May be useful to add expected input format as it took me a minute to figure out why u kept skipping the first index LOL

Reply:
How do I add this? Like a comment?

Reply:
In this case, yeah. Not sure how to do proper defenitions in python